litmus/sched_pres.c

```diff
@@ -608,5 +609,11 @@ static int __init init_pres(void)
 	return register_sched_plugin(&pres_plugin);
 }
 
-module_init(init_pres);
+static void __exit exit_pres(void)
+{	
+	if (unregister_sched_plugin(&pres_plugin))
```

Comment:
What if this call fails because PRES is the active plugin and there are active real-time tasks?

Reply:
Answered at the pull request conversation